src/pci.rs

```diff
@@ -107,6 +107,30 @@ struct PciBar {
     address: u64,
 }
 
+#[cfg(not(test))]
+enum PciCapabilityType {
```

Comment:
Those types are really virtio based. That's why I would call this enum `VirtioPciCapabilityType`.

Following the same observation, I think we should put those into their own separate file `virtio-pci.rs`. A clear separation between PCI and virtio-pci seems important to me.

Reply:
Agreed to the latter. But i'm not going to split the file at this point as PCI is only used for virtio-pci.